### ISSUE TITLE
Top of letters with diacritical marks are cut off in some cases.

### ISSUE THREAD

**Issue by zonkmachine** [pioneerspacesim/pioneer#5476]
Letters on the first BBS post have their top part cut off if they stick up above the rest such as with diacritical marks. Example below is the Swedish letter **'Ä'** being decapitated. This seem to be general of the text on any first row on a page. Another example being the name of the station in the lobby. Swap out Cydonia in data/systems/00_sol.lua to 'Nuevo Östersund'.

### Observed behaviour
![BBS-top-margin](https://user-images.githubusercontent.com/6368949/213008901-8d698ed4-9475-470f-851b-a06d3efd638d.png)

__My pioneer version (and OS):__

```
git show
commit e3a86ef82c44d707fdc04ea3581991642d905779 (HEAD -> master, upstream/master, upstream/HEAD)
Author: Pioneer Transifex <pioneer-transifex@pioneerspacesim.net>
Date:   Tue Jan 17 03:01:20 2023 +0100

    auto-commit: translation updates
```

Release 22.04.1 LTS (Jammy Jellyfish) 64-bit
Kernel Linux 5.15.0-58-generic x86_64

**Comment by impaktor** [comment]
Good it's not an "AFFÄRSRESA TILL HÖRBY"....
(Swedish joke)

**Comment by zonkmachine** [comment]
Update: It's more general and not just the BBS. I changed the name of Cydonia to Östersund and the lovely dots over the O was cut off in the lobby. I also changed the line in the Repair shop, "Your ship is in perfect working condition" to have capital letters with dots ([umlaut marks) and sure enough, cut off.
![ostersund](https://github.com/pioneerspacesim/pioneer/assets/6368949/14a2d739-6535-4703-b585-1075d5d0220c)

**Comment by impaktor** [comment]
Do we have umlaut for the font used in the headline? @nozmajner

**Comment by zonkmachine** [comment]
Yes. As you see in the first example from the bbs the second post shows Ä correctly.

**Comment by impaktor** [comment]
I don't think it's the same font used in BBS advert listing, as in the lobby, regarding "Östersund".

**Comment by zonkmachine** [comment]
Maybe not. A lowercase ö still shows up if I change name of Cydonia to Östersönd.

**Comment by nozmajner** [comment]
Both Pionillium and Inpionata has Ää

**Comment by zonkmachine** [comment]
Yes, but it's not the font. As you can see there is umlaut enough in the fonts but on a top line, if it's an uppercase letter, they are cut off. It's like there isn't enough space reserved for the top of the glyph.

**Comment by zonkmachine** [comment]
I get a similar issue in Libreoffice. Top line cuts off the umlaut stuff in Orbitron but not in Times New Roman or Arial. It looks like it could be the font/s after all.

**Comment by impaktor** [comment]
Could just use danish alphabet instead, with "aa" = å. Solves all the umlaut issues ;)

**Comment by zonkmachine** [comment]
> Could just use danish...

No. :sweden: 

I've managed to fire up a font editor and tweak Orbiteer a bit to coerce the dots to appear. Font editing may not be the best way to do this though. I tried Abiword and it seem to know of this kind of borked fonts and loads them fine.

**Comment by zonkmachine** [comment]
It turned out to be pretty simple to fix this in BirdFont by just editing the top/bottom limits. I nudged them all up so the size of the font wouldn't change and  this is what came out of it. Parts of the text look better ('Lobby' and 'Tanka fullt', in my opinion) and some look a bit less good. The date/clock I think looked better in the old position for instance.' Still looking for a fix in the code though.

Current:
![font1](https://github.com/pioneerspacesim/pioneer/assets/6368949/894547fb-3a84-444a-a117-529791c1974f)

Fixed:
![font2](https://github.com/pioneerspacesim/pioneer/assets/6368949/b86663e7-a730-4262-8220-eda20e20e1a5)

**Comment by zonkmachine** [comment]
Orbitron (Orbiteer) has updated forks. I tried [the one at Google Fonts](https://github.com/googlefonts/orbitron-vf) that seem most to have been updated with a variable font. The static fonts seem to have been updated too, probably covering more glyphs. It unfortunately didn't solve this issue. Maybe bump this up-stream somewhere? The original source hasn't been updated in some 14 years. I'll see if my fix is still good.

**Comment by zonkmachine** [comment]
<img width="1186" height="457" alt="Image" src="https://github.com/user-attachments/assets/0dc820b9-72d6-444a-9bb1-725e7773b2e9" />

Orbiteer/Orbitron and Pionillium both show issues when text is selected when working in a text editor. They both also show artifacts when editing text that goes away when scrolling but not when typing. Here are the two together with Times New Roman. All other tested fonts (Arial, Noto Sans, etc..) correctly selects all of the letters. Tested editors, Abiword and LibreOffice.

**Comment by zonkmachine** [comment]
Pushed a fix that adds padding. I can't stomach editing the actual fonts right now.
